docs/content/guides/2.alokai-essentials/1.alokai-next-js/9.udl.md

```diff
@@ -62,7 +62,9 @@ The `unifiedApiExtension` object is the result of calling `createUnifiedExtensio
 
 #### transformImageUrl
 
-In the [Connecting Product Details Page with SAP Commerce Cloud](./7.connecting-pdp.md) guide, we used `transformImageUrl` method to transform the image URL. It's not recommended to do this in the `ProductDetails` component. Instead, we can pass the `transformImageUrl` method to the `unifiedApiExtension` object as config, so that this transformation will happen in the Middleware.
+SAP Commerce Cloud stores image urls as relative paths but the storefront needs absolute paths to display images. `transformImageUrl` function is responsible for providing correct image url.
```

Comment:
A quick explanation of what `transformImageUrl` does would help. Does it add the base path? How is this base path configured? Just a one sentence of what it does to eliminate any confusion.